### ISSUE TITLE
Wrong model and return order

### ISSUE THREAD

**Issue by mnchapel** [alzayats/DeepFish#15]
Hi, I got two questions about your code.

The first one is about the model used for the clf and reg task. In the exp_config.py file, there are two models for these tasks: inception and resnet. When I use your code, the inception model is used but it is not the one indicated in your paper. So I just need to remove inception from the exp_config.py file to use resnet (which seems to be the right one)?

Also in the code of resnet.py, in the forward function, you take the logits from the backbone in that order "logits_32, logits_16, logits_8" but if you take a look at the backbone the return of the extracted_features function is "x_8s, x_16s, x_32s". Is that normal? or I am missing something?

Thank you

**Comment by alzayats** [comment]
Hello,

Thank you for your questions and interest in the DeepFish project. 

**Regarding the model used for the clf and reg tasks:** Yes, you’re correct. The paper mentions the use of ResNet, not Inception. If you want to use ResNet as specified in the paper, you should indeed remove or comment out the Inception model from the exp_config.py file. This will ensure that the ResNet model is used for your tasks.

**Regarding the order of logits in the forward function of resnet.py:** The issue you pointed out has been addressed and fixed.

I hope this helps! If you have any more questions or need further clarification, please don’t hesitate to ask

**Comment by mnchapel** [comment]
Thank you very much for your quick reply!
